string scenarioName
string[] nodes
NodeLaunchParam[] launchparams
---
bool res
